src/image/descriptor.rs

```diff
@@ -0,0 +1,102 @@
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+
+make_pub!(
+    #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+    #[cfg_attr(
+        feature = "builder",
+        derive(derive_builder::Builder, getset::Getters),
+        builder(pattern = "owned", setter(into, strip_option)),
+        getset(get = "pub")
```

Comment:
This has to be removed, otherwise we will get a getter and a copygetter for `size: i64`:
```suggestion
```